element-connector/src/main/java/org/eclipse/californium/elements/RawData.java

```diff
@@ -330,4 +330,16 @@ public boolean isSecure() {
 		return (correlationContext != null &&
 				correlationContext.get(DtlsCorrelationContext.KEY_SESSION_ID) != null);
 	}
+
+	/**
+	 * Callback, when context gets available. Used on sending an message.
+	 * 
+	 * @param context established context to be forwarded to the callback.
+	 */
+	public void onContextEstablished(CorrelationContext context) {
```

Comment:
My understanding is that this method should be used instead of `getMessageCallback().onContextEstablished()`. I guess `DTLSConnector` uses the latter and should probably be changed to instead call this new method, shouldn't it?